docs/GettingStarted.rst

```diff
@@ -0,0 +1,86 @@
+.. raw:: html
+
+   <!-- This Source Code Form is subject to the terms of the Mozilla Public
+      - License, v. 2.0. If a copy of the MPL was not distributed with this
+      - file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+===============
+Getting Started
+===============
+Get started by installing Frost and running some pre-built tests.
+
+Installing Frost
+================
+
+Clone the repo::
+
+   git clone git@github.com:mozilla/frost.git
+
+Create a virtual environment::
+
+   cd frost
+   python3 -m venv venv  
+
+Activate the virtual environment::
+
+   . venv/bin/activate
+
+Upgrade pip::
+
+   pip install --upgrade pip
+
+Install Requirements::
+
+   pip install -r requirements.txt
+
+Running a pre-built AWS test
+============================
+
+Pre-requirements
+----------------
+
+Frost uses the `boto 3 <https://github.com/boto/boto3/>`_ library to communicate with AWS. Prior to running tests you must `configure <https://boto3.amazonaws.com/v1/documentation/api/latest/guide/quickstart.html#configuration>`_ your environment authentication credentials to work with boto3, and then authenticate to AWS.  See the AWS `documentation <https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-welcome.html>`_ for assistance.
+
+Pick a test
+-----------
+
+The tests for AWS live under the associated services directories in the aws directory.
+
+Example run::
+
+   pytest aws/elb/test_elb_instances_attached.py
```

Comment:
Yes, AND there are often options you want to specify.  Maybe not appropriate in the getting started beyond a note to "see X for available options".

I'm not up on the details of running AWS tests, but I believe there are ways to specify which account(s) to examine.

Reply:
Before writing this I re-read the Sphinx docs, and the foxsec-automation docs, and did not come back to the README here.  I'll check back on this tomorrow, but I'll probably just scrap this whole PR.

Reply:
Does this suggest there are too many places things are documented?

@hwine do you think we need both an extensive readme and the sphinx docs, or could we merge most of the readme into sphinx and link to it there?

Reply:
We'll chat at today's session, but tl;dr I plan to include the README in the rendered docs. (some folks cringe at 2-screenful functions, but think 700 line docs don't need refactoring) :D :D